src/frontend/web_application/src/components/ContactDetails/index.jsx

```diff
@@ -74,22 +87,22 @@ class ContactDetails extends Component {
           && remoteIdentity.identity_id === identityId);
   }
 
-  renderSubtitleActions() {
+  renderSubtitleActions(panel) {
     const { __ } = this.props;
-    const activeButtonProp = this.state.editMode ? { color: 'active' } : {};
+    const activeButtonProp = this.state.editMode[panel] ? { color: 'active' } : {};
 
     return (
       <Button
```

Comment:
@ dey there is a bug here: Button doesn't responds to click when you click on the inner `<Icon />`. If you click anywhere else on the Button, it works. I played with "pointer-event" css on the Icon, but doesn't solve the issue.
I think it's because `value` attribute on Button is not null (that's the only difference with other Buttons that work fine), but can't fix that.